lib/matplotlib/patches.py

```diff
@@ -2140,6 +2115,44 @@ def segment_circle_intersect(x0, y0, x1, y1):
         # restore original path
         self._path = path_original
 
+    def _update_path(self):
+        # Compute new values and update and set new _path if changed
+        stretched = self._theta_stretch()
+        if any(a != b for a, b in zip(stretched, (self._theta1, self._theta2,
+                                                  self._width, self._height))):
+            self._theta1, self._theta2, self._width, self._height = stretched
```

Comment:
How much work would it be to add a test that exercises updating an Arc?

Reply:
I thought about it and there are two things to consider here:
1. Is is actually a use case that someone creates an Arc and then sets e.g. theta1 to a new value? (I haven't really considered that.)
2. What I think one would like to test is to change the size. I was a bit worried that it wouldn't work with the collection, but surprisingly it did (manually tested with resizing). I cannot really explain why, but probably because I do not fully understand the code. (Otherwise I expected that the special draw method would not be called for collections, so the adjusted theta values would not be updated, but somehow it works...) However, I do not really know how to write such a test, but maybe it is enough to just create a figure with a given size and then resize it? Will be easy to see if we hit that line anyway...